hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServerSource.java

```diff
@@ -316,6 +316,8 @@ public interface MetricsRegionServerSource extends BaseSource, JvmPauseMonitorSo
   String BLOCK_CACHE_FREE_DESC = "Size of the block cache that is not occupied.";
   String BLOCK_CACHE_COUNT = "blockCacheCount";
   String BLOCK_CACHE_COUNT_DESC = "Number of block in the block cache.";
+  String BLOCK_CACHE_DATA_BLOCK_COUNT = "blockCacheDataBlockCount";
```

Comment:
I also added this metric because I noticed the data was available, and this should be helpful for people looking to figure out how to size their L1 vs L2 cache. The L1 cache ends up holding only non-DATA blocks, so you could estimate a good size for L1 by doing `(blockCacheCount - blockCacheDataBlockCount) * blockSize`. This should give a rough estimate of the amount of META and BLOOM_CHUNK blocks that would need to fit in the L1.

Reply:
Add a small doc patch to note this?